cocos/audio/AudioEngine.cpp

```diff
@@ -378,6 +404,22 @@ void AudioEngine::uncache(const std::string &filePath)
     auto audioIDsIter = _audioPathIDMap.find(filePath);
     if (audioIDsIter != _audioPathIDMap.end())
     {
+	#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+        auto lst =  _audioPathIDMap[filePath];
```

Comment:
- audioIDsIter contains the result, so can just use it
- and if still need to use it like this, i think it is better to use const reference to improve performance

Reply:
recovered